### PR TITLE
Log deployed contracts and a Flag to turn on fees

### PR DESCRIPTION
**Pull request by janezpodhostnik**
ref: https://github.com/dapperlabs/flow-internal/issues/1384

## Description

There are two changes here:
- New flag to turn on transaction fees
- Added logging the addresses of the deployed flow contracts (see image)

image

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: blockchain.go [hunk 1]
```diff
@@ -422,17 +432,26 @@ func bootstrapLedger(
...
-	if ctx.LimitAccountStorage {
-		return fvm.Bootstrap(flowAccountKey,
+func configureBootstrapProcedure(conf config, flowAccountKey flowgo.AccountPublicKey, supply cadence.UFix64) *fvm.BootstrapProcedure {
+	options := make([]fvm.BootstrapProcedureOption, 0)
+	options = append(options,
+		fvm.WithInitialTokenSupply(supply),
+	)
+	if conf.StorageLimitEnabled {
+		options = append(options,
 			fvm.WithInitialTokenSupply(supply),
```
Comment (turbolent) [inline blockchain.go:442]: Already added above (https://github.com/onflow/flow-emulator/pull/28/files#diff-1fb2891d150a66e1319495e4f10a7ce04855476836a3a6b17f253e312f37d23bR438)

```suggestion
```

File: blockchain.go [hunk 2]
```diff
@@ -524,6 +543,10 @@ func (b *Blockchain) getBlockByHeight(height uint64) (*flowgo.Block, error) {
 	return block, nil
 }
 
+func (b *Blockchain) GetChain() flowgo.Chain {
+	return b.vmCtx.Chain
```
Comment (turbolent) [inline blockchain.go:547]: 👌

File: server/server.go [hunk 3]
```diff
@@ -113,6 +115,17 @@ func NewEmulatorServer(logger *logrus.Logger, conf *Config) *EmulatorServer {
...
 
+	chain := blockchain.GetChain()
+
+	contracts := logrus.Fields{
+		"FlowServiceAccount": chain.ServiceAddress().HexWithPrefix(),
+		"FlowToken":          fvm.FlowTokenAddress(chain).HexWithPrefix(),
+		"FungibleToken":      fvm.FungibleTokenAddress(chain).HexWithPrefix(),
+		"FlowFees":           fvm.FlowFeesAddress(chain).HexWithPrefix(),
+		"FlowStorageFees":    chain.ServiceAddress().HexWithPrefix(),
+	}
```
Comment (turbolent) [inline server/server.go:126]: 😍

**Review by turbolent — APPROVED** [review]
Totally missed this, nice!

File: blockchain.go (hunk 1 above)
Comment (janezpodhostnik) [reply blockchain.go:442]: whoops, missed that one!

**Pull request merged**